Remove memprof from dockerfile (#1739)

phalene-bytes opened the pull request:
Docker images should now build successfully

melaniekung reviewed (changes requested):
hi eleanor, this fix looks good to me! but before we merge, can you provide a little more details on this fix in the commit message? was it really a typo or a dependency req for an image we're using in docker? also, a little background on what this pkg is used for, that would also be great. thank you!

phalene-bytes commented:
@melaniekung Mmm what do you think I should put?  After the fix, the rest of the Dockerfile commands are able to run (like building the B2 & B5 themes). Should I put:

Fix: Typo in Dockerfile (`php8-pecl-memprof` should be `php81-pecl-memprof`)
This dependency is a memory profiler for PHP, used to identify memory leaks in PHP scripts. The typo caused a failure in the Docker image build process. The fix now enables the remaining commands in the Dockerfile to run successfully.

For more information on this dependency, see: https://github.com/artefactual/atom/pull/1452

melaniekung commented:
i couldn't confirm if it was a typo or if the difference is due to support for php 8 vs php 8.1 since docker used to build fine. if my assumption is correct, then i'd say something like "Update pecl-memprof dependency, a memory profiler for PHP used to identify memory leaks in PHP scripts". otherwise "Fix typo for pecl-memprof dependency, a memory profiler for PHP used to identify memory leaks in PHP scripts" will do.

phalene-bytes committed:
Remove memprof from dockerfile (#1739)

The module 'php8-pecl-memprof' has changed to 'php81-pecl-memprof'. It is a memory profiler for PHP, used to identify memory leaks in PHP scripts. The renaming caused a failure in the Docker image build process.
For more info on this module, see: https://github.com/artefactual/atom/commit/3745f04d21180456b8b4d4520fae5cd21463ead5
